src/main/java/de/tum/in/www1/artemis/service/dto/GradingCriterionDTO.java

```diff
@@ -0,0 +1,22 @@
+package de.tum.in.www1.artemis.service.dto;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.validation.constraints.NotNull;
+
+import de.tum.in.www1.artemis.domain.GradingCriterion;
+
+public record GradingCriterionDTO(long id, String title, Set<GradingInstructionDTO> structuredGradingInstructions) {
```

Comment:
I think they can, but it would not make any difference in how Athena would process it 🤔 
I'll add the line regardless because it seems useful for consistency and simpler debugging of Athena in the future.